[#503] Customize commit interval so the test environment can have a d… (#555)
airyhq/airy: docs/docs/guides/airy-core-in-production.md

However, we provide a way to deploy the whole Kafka cluster on top of Kubernetes
with Helm as we use this approach for test installations.

The default commit interval is set to 1000 ms (1 second). This is *not* recommended
for production usage.

> You change the default value to a more suitable amount on the configuration file

Reviewer: ```suggestion
You change the default value to a more suitable value on the configuration file
```

> You change the default value to a more suitable amount on the configuration file
> `infrastructure/helm-chart/charts/apps/charts/airy-config/values.yaml` by changing
> the `commitInterval` value.

Reviewer: ```suggestion
You change the `commitInterval` to a more suitable production value in the configuration file
`infrastructure/helm-chart/charts/apps/charts/airy-config/values.yaml`.
```
